mutacc/builds/build_case.py

```diff
@@ -45,7 +46,7 @@ def __init__(self, input_case, read_dir, padding=None, picard_exe=None):
         # Build case
         self['case'] = self.input_case['case']
 
-    def _build_variants(self, padding=None, rank_model_version=None):
+    def _build_variants(self, padding=None, rank_model_version=None, vcf_parse=None):
```

Comment:
Same here, be more explicit in the docstring. Is rank_model_version str or float etc